frontend/src/router.js

```diff
@@ -23,10 +23,8 @@ export default new Router({
     {
       path: '/question/:id',
       name: 'question',
-      props: ({params: { id }}) => ({
-        id
-      }),
-      component: () => import(/* webpackChunkName: "about" */ './views/Question.vue')
+      props: ({params: { id }}) => { id },
```

Comment:
This is a bug as it stands.
() => {} // doesn't return an object
() => ({}) returns an empty object

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
oic